File: GDGLimaApp/app/src/main/java/com/gdglima/glabkotlin/gdglimaapp/ui/adapter/RecyclerClickListener.java
package com.gdglima.glabkotlin.gdglimaapp.ui.adapter;

import android.view.View;

/**
 * Created by emedinaa on 14/10/15.
 */
public interface RecyclerClickListener {
    public void onClick(View view, int position);

    public void onLongClick(View view, int position);
}

File: GDGLimaApp/app/src/main/java/com/gdglima/glabkotlin/gdglimaapp/ui/adapterskt/RecyclerClickListener.kt
package com.gdglima.glabkotlin.gdglimaapp.ui.listener

import android.view.View

/**
 * Created by emedinaa on 14/10/15.
 */
interface RecyclerClickListener {
    fun onClick(view: View, position: Int)

    fun onLongClick(view: View, position: Int)
}